pkg/liquidity-mining/contracts/gauges/SingleRecipientGaugeFactory.sol

```diff
@@ -52,11 +67,14 @@ contract SingleRecipientLiquidityGaugeFactory {
      * @return The address of the deployed gauge
      */
     function create(address recipient) external returns (address) {
+        require(_recipientGauge[recipient] == address(0), "Gauge already exists");
```

Comment:
This seems like an odd restriction, though I don't see why we'd have gauges with the same type and same recipient (why not use the same one?), or different type and same recipient (since each type is quite different).

Reply:
Yeah, I was slightly torn on this.

Gauges are defined by a "key"-gauge mapping and for the standard L1 and L2 gauges this mapping is naturally 1-to-1 as you can't stake into multiple gauges and two root gauges bridging funds to a single streamer is equivalent to one. 

`SingleRecipientGauge` is the only situation where it kinda makes sense to relax this to be 1-to-many. Example of this being the temporary gauges for veBAL/Polygon/Arbitrum gauge types can't just send funds to the LM committee multisig but need to be proxied through another contract/multisig.

I'm erring on the side of maintaining 1-to-1 for reasons of consistency, discouraging commingling of funds where possible and an ability to look up the gauge for a given recipient easily.

Reply:
That said, this change is already in master.

Reply:
I think those trade-offs make sense. Thanks!